samples/golang/gorm/create_data_model.sql

```diff
@@ -1,11 +1,17 @@
 -- Executing the schema creation in a batch will improve execution speed.
-start batch ddl;
+-- All statements that are prefixed with /* skip_on_open_source_pg */ will be skipped on open-source PostgreSQL.
+
+/* skip_on_open_source_pg */ start batch ddl;
 
 create table if not exists singers (
     id         varchar not null primary key,
     first_name varchar,
     last_name  varchar not null,
-    full_name  varchar generated always as (coalesce(concat(first_name, ' '::varchar, last_name), last_name)) stored,
+    full_name varchar(300) generated always as (
+        CASE WHEN first_name IS NULL THEN last_name
+             WHEN last_name  IS NULL THEN first_name
+             ELSE first_name || ' ' || last_name
+        END) stored,
```

Comment:
This needed to be changed, as `concat` is not an `IMMUTABLE` function on open-source PostgreSQL, which again means that it is not allowed for a generated column. The `||` operator is considered `IMMUTABLE`.